comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -19,42 +67,152 @@ public static long GetFileSize(string filePath)
 		public static bool IsDirectory(string path)
 		{
 			return File.GetAttributes(path).HasFlag(FileAttributes.Directory);
-		}
-
-		// Return the total size, in bytes, of all the files below the given directory
-		public static long GetTotalSize(string directory)
+		}             
+     
+        #region Get Total File Size
+        // Return the total size, in bytes, of all the files below the given directory
+        public static long GetTotalSize(string directory)
 		{
-			throw new NotImplementedException();
+            directoryData = new DirectoryData(directory);
+            return GetTotalFileSize(directoryData.filePaths);
 		}
 
-		// Return the number of files (not counting directories) below the given directory
-		public static int CountFiles(string directory)
+        /// <summary>
+        /// Uses all of the files and gets their sizes and then adds them up.
+        /// </summary>
+        /// <param name="filePaths"></param>
+        /// <returns>
+        /// The file size of all of the file paths provided by the parameters.
+        /// </returns>
+        private static long GetTotalFileSize(string[] filePaths)
```

Comment:
Naming doesn't make clear the difference between `GetTotalSize` and `GetTotalFileSize`